_data/simple-icons.json

```diff
@@ -3410,6 +3410,11 @@
             "hex": "FCC624",
             "source": "http://www.linuxfoundation.org/about/about-linux"
         },
+        {
+            "title": "Linux Containers",
+            "hex": "333333",
+            "source": "https://discuss.linuxcontainers.org/t/official-lxc-lxd-svg-icon/7346"
```

Comment:
Why not linking to https://linuxcontainers.org/static/img/containers.svg?

Reply:
Rebuilding from source and removing some of the weird paths, I got this:

![linuxcontainers](https://user-images.githubusercontent.com/1830867/103231873-d40fdf80-4930-11eb-97d8-fe1dd4918a90.png)

```svg
M10.15,8.98l1.647,0.951l-5.756,3.345l0.005-1.911L10.15,8.98z M11.873,7.979L10.32,8.881l1.548,0.893
	L11.873,7.979z M6.028,5.33L6.025,6.4l0.543,0.316l3.602,2.079l1.632-0.948L9.612,6.568L6.381,4.681L6.03,4.478L6.028,5.33z
	 M6.1,4.347l0.359,0.209l6.321,3.65l5.258,3.037l5.858-3.405l-11.94-6.895L6.1,4.347z M12.102,16.949l-0.005,1.924l5.858-3.404
	l0.005-1.924L12.102,16.949z M12.025,7.94L12.02,9.862l5.94,3.428l0.005-1.92L12.025,7.94z M1.895,8.885l4.075,2.352l4.031-2.342
	L5.926,6.542L1.895,8.885z M24,7.982l-5.858,3.404l-0.015,3.982l5.858-3.404L24,7.982z M11.952,18.022l0.003-1.073L7.6,14.436
	l-1.565-0.903v0.001l-0.939-0.542l-5.081-2.932L0.01,11.979l11.94,6.895L11.952,18.022z M17.887,13.417l-5.922-3.411l-5.853,3.401
	l5.917,3.414L17.887,13.417z M23.959,12.179l-11.862,6.864l-0.01,4.013l11.863-6.894L23.959,12.179z M11.944,21.27l0.005-2.227
	L0.01,12.148L0,16.162l11.94,6.895L11.944,21.27z M0.021,9.802l1.579-0.917L0.025,7.976L0.021,9.802z M5.853,6.412l0.024-1.636
	l0.001-0.296L0.099,7.848l1.647,0.951L5.853,6.412z M5.894,11.363L1.749,8.97l-0.46,0.267L0.094,9.932l5.795,3.345L5.894,11.363z
```

Feel free to use it, or I can open a PR for it?

Reply:
@runxel, we try to link to a page rather than a file whenever possible as pages are less prone to link-rot.

Reply:
@PeterShaggyNoble - the source of their website, including this SVG, is available at https://github.com/lxc/linuxcontainers.org/blob/29d3299ddf8718099b6de1464570fbbadbaabecb/static/img/containers.svg

Reply:
Yep, definitely, @adamrusted :+1: Just pushed that change.